editor/src/components/navigator/left-pane.tsx

```diff
@@ -92,17 +106,203 @@ export const LeftPaneComponent = betterReactMemo('LeftPaneComponent', () => {
           }
         }}
       >
-        {selectedTab === LeftMenuTab.Project ? <ProjectPane /> : null}
+        {!isMyProject ? <ForksGiven /> : null}
```

Comment:
Double negatives. :(

Reply:
however, all of the lines above and below have the <Component> : null structure, I tried null : <ForksGiven /> here and that seemed worse